allel/util.py

```diff
@@ -145,6 +146,48 @@ def ensure_square(dist):
     return dist
 
 
+def mask_inaccessible(is_accessible, pos, *arrays):
+    """
+    This function returns a tuple (positions, *arrays) in which
+    positions that are not accessible are removed from the positions
+    and the *arrays.
+
+    Parameters
+    ----------
+    is_accessible : array_like, bool, shape (len(contig),)
+        Boolean array indicating accessibility status for all positions in the
+        chromosome/contig.
+    array1, array2, ...  : array_like
+        N-dimensional array objects with n_variants elements in the 1D.
+    pos : array_like, int, shape (n_variants,)
+        Variant positions, using 1-based coordinates, in ascending order.
```

Comment:
```suggestion
    pos : array_like, int, shape (n_variants,)
        Variant positions, using 1-based coordinates, in ascending order.
    array1, array2, ...  : array_like
        N-dimensional array objects with n_variants elements in the 1D.
```

Flip order to match arg order.